### ISSUE TITLE
README says supports Swift 3 even though SDK doesn't

### ISSUE THREAD

**Issue by theodoracheng** [openwhisk/openwhisk-client-swift#32]
The README currently says `The client SDK is compatible with Swift 3.x and runs on iOS 10, WatchOS 3, and Darwin.`

I'm using Xcode Version 8.0 (8A218a) and did a pod install with

```
     pod 'OpenWhisk', :git => 'https://github.com/openwhisk/openwhisk-client-swift.git', :tag => '0.2.1'
```

However, I'm getting this error.

```
[!] Unable to satisfy the following requirements:

- `OpenWhisk (from `https://github.com/openwhisk/openwhisk-client-swift.git`, tag `0.2.1`)` required by `Podfile`

Specs satisfying the `OpenWhisk (from `https://github.com/openwhisk/openwhisk-client-swift.git`, tag `0.2.1`)` dependency were found, but they required a higher minimum deployment target.
```

Does the SDK not support Swift 3? Additionally, can supported versions of Xcode be added into the release description?
- [ ] Update README with more detail on what is supported in Swift 3 and what versions
- [ ] Update releases with more clear description on what it includes, what versions are being supported

**Comment by paulcastro** [comment]
@theodoracheng , the SDK should support Swift 3 and Xcode 8.

What is the deployment target of your Xcode project?  The SDK is set for iOS 10.

I can make a new version that supports iOS 9 if thats a requirement for you.

**Comment by theodoracheng** [comment]
@paulcastro yes, if it's not too much trouble to support iOS9 that would be great. 
I appreciate your quick reply.

After setting it to iOS 10 my pod successfully installed.

**Comment by paulcastro** [comment]
Glad it worked - it makes sense to support iOS 9 as a deployment target.  I'll push a new version and close this issue when its done.

**Comment by theodoracheng** [comment]
@paulcastro thanks. Could you also put a description in the new release stating the new feature? That way anyone can come in and look to see what's supported in what versions.

**Comment by paulcastro** [comment]
Fixed in PR #33
